src/main/java/org/folio/rest/impl/PermsAPI.java

```diff
@@ -428,7 +422,7 @@ public void deletePermsUsersById(String userid, Map<String, String> okapiHeaders
                     return;
                   }
                   pgClient.delete(connection, TABLE_NAME_PERMSUSERS,
-                      new Criterion(idCrit), deleteReply -> {
+                      idCrit, deleteReply -> {
```

Comment:
```suggestion
                      userid, deleteReply -> {
```

Reply:
OK. Not that I think it improves readability.. idCrit is the one we are dealing with and when idCrit is using correct query type, it should be the same.